mne/viz/epochs.py

```diff
@@ -442,6 +443,229 @@ def plot_epochs(epochs, epoch_idx=None, picks=None, scalings=None,
     return fig
 
 
+def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20,
+                n_channels=10, bad_color=(1.0, 0.0, 0.0),
+                fig_title=None, show=True, block=False):
+    """ Visualize single trials.
+
+    Bad epochs can be marked with a left click on top of the epoch.
+    Calling this function drops all the selected bad epochs as well as bad
+    epochs marked beforehand with rejection parameters.
+    The scaling can be adjusted with 'page up' and 'page down'.
+
+    Parameters
+    ----------
+
+    epochs : instance of Epochs
+        The epochs object
+    picks : array-like of int | None
+        Channels to be included. If None only good data channels are used.
+        Defaults to None
+    scalings : dict | None
+        Scale factors for the traces. If None, defaults to:
+        `dict(mag=1e-12, grad=4e-11, eeg=20e-6, eog=150e-6, ecg=5e-4, emg=1e-3,
+             ref_meg=1e-12, misc=1e-3, stim=1, resp=1, chpi=1e-4)`
+    n_epochs : int
+        The number of epochs per view. Defaults to 20.
+    n_channels : int
+        The number of channels per view. Defaults to 10.
+    bad_color : Tuple
+        A matplotlib-compatible color to use for bad channels. Defaults to
+        (1.0, 0.0, 0.0) (red).
+    fig_title : str | None
+        The title of the window. If None, epochs name or <unknown> will be
```

Comment:
If it's a list I get `<unknown>`. This is too vague to me. Why not say: "List of 4 epoch objects" if there are 4 epochs and you don't want to spell them all out?